salt/kubernetes-master/deploy_addons.sh

```diff
@@ -25,15 +25,36 @@ else
 fi
 
 echo "Deploying DNS on Kubernetes"
-KUBEDNS=`eval "$KUBECTL_PATH $NS_ARGS $SRV_ARGS get rc | grep kube-dns | cat"`
+
```

Comment:
I know this is all existing code, however, we really should be doing all this from salt directly, rather than from a bash script.